app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,16 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // force interger input and submit on enter
+      var x = event.which;
+      if ((x > 8 && x < 48) || (x > 57 && x < 96) || x > 105) {
```

Comment:
> Then something along the lines of:

As @vladikoff noted, you'll have to allow the `ENTER` key event to pass through unscathed.